.github/workflows/ci.yml

```diff
@@ -961,14 +961,23 @@ jobs:
           name: ${{ steps.constants.outputs.external_result_name }}
           path: external-result
 
-      - name: Check
+      - name: Debug out
```

Comment:
What is this? Maybe improve the name if this is something that we should look at?

Reply:
done.